rust_src/src/crypto/mod.rs

```diff
@@ -63,205 +52,6 @@ fn hash_alg(algorithm: LispSymbolRef) -> HashAlg {
     }
 }
 
```

Comment:
This was all waiting for `extract_data_from_object` to be ported. No sense keeping this zombie alive.